kluent-1.69 is configured for jvm version 17

Pulling in kluent 1.69 into a project that builds against a java version less than 17 currenlty breaks with the following message:

<details><summary>Stacktrace</summary>
<pre>
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':kris-core:test'.
> Could not resolve all task dependencies for configuration ':kris-core:testRuntimeClasspath'.
   > Could not resolve org.amshove.kluent:kluent:1.69.
     Required by:
         project :kris-core
      > No matching variant of org.amshove.kluent:kluent:1.69 was found. The consumer was configured to find a runtime of a library compatible with Java 11, packaged as a jar, preferably optimized for standard JVMs, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm' but:
          - Variant 'apiElements' capability org.amshove.kluent:kluent:1.69 declares a library, packaged as a jar, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm':
              - Incompatible because this component declares an API of a component compatible with Java 17 and the consumer needed a runtime of a component compatible with Java 11
              - Other compatible attribute:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
          - Variant 'javadocElements' capability org.amshove.kluent:kluent:1.69 declares a runtime of a component, and its dependencies declared externally:
              - Incompatible because this component declares documentation and the consumer needed a library
              - Other compatible attributes:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
                  - Doesn't say anything about its target Java version (required compatibility with Java 11)
                  - Doesn't say anything about its elements (required them packaged as a jar)
                  - Doesn't say anything about org.jetbrains.kotlin.platform.type (required 'jvm')
          - Variant 'runtimeElements' capability org.amshove.kluent:kluent:1.69 declares a runtime of a library, packaged as a jar, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm':
              - Incompatible because this component declares a component compatible with Java 17 and the consumer needed a component compatible with Java 11
              - Other compatible attribute:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
          - Variant 'sourcesElements' capability org.amshove.kluent:kluent:1.69 declares a runtime of a component, and its dependencies declared externally:
              - Incompatible because this component declares documentation and the consumer needed a library
              - Other compatible attributes:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
                  - Doesn't say anything about its target Java version (required compatibility with Java 11)
                  - Doesn't say anything about its elements (required them packaged as a jar)
                  - Doesn't say anything about org.jetbrains.kotlin.platform.type (required 'jvm')
</pre>
</details>

The root cause is most likely that attribute variants.attributes."org.gradle.jvm.version" in `kluent-1.69.module` is set to `17`:
![image](https://user-images.githubusercontent.com/357238/195947850-b7c4dd8f-ab95-42a9-891f-effafd6b5960.png)

The java version compatibility for the module declaration should presumably be set to 1.6/1.8 as you already do e.g. for tasks of type `compileKotin` setting jvmTarget.

--- a/build.gradle
+++ b/build.gradle
@@ -68,6 +68,10 @@ subprojects {
             withSourcesJar()
         }
 
+        def isAndroid = project.rootProject.hasProperty("ANDROID")
+        sourceCompatibility = isAndroid ? 1.6 : 1.8
+        targetCompatibility = isAndroid ? 1.6 : 1.8
+
         publishing {
             repositories {
                 maven {

--- a/common/build.gradle
+++ b/common/build.gradle
@@ -10,6 +10,9 @@ dependencies {
 
 configurePublishing()
 
+sourceCompatibility = 1.6
+targetCompatibility = 1.6
+
 compileKotlinCommon {
     kotlinOptions {
         freeCompilerArgs += [

